### PR TITLE
Update package.json for just-animate

### PR DESCRIPTION
**Pull request by notoriousb1t**
- updated website link
- updated folder structure; will be moved to dist from browser folder when I publish 1.1.0
- changed filename to just-animate-all.min.js (the current filename doesn't point to the actual lib)

### CODE REVIEW

File: ajax/libs/just-animate/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "just-animate",
-  "filename": "just-animate-animations.min.js",
+  "filename": "just-animate-all.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/just-animate/package.json]: Please don't touch filename here unless you added a version with the new name. I can take care of this change, thanks.

Comment (notoriousb1t) [inline ajax/libs/just-animate/package.json]: For some reason I thought it would auto pull from NPM :)  I'll amend this with the 1.0.4 files

**Comment by notoriousb1t** [comment]
I'm going to resubmit this in a new PR.  Sorry for the trouble so far.

**Comment by PeterDaveHello** [comment]
@notoriousb1t No problem, don't worry :)

**Pull request closed without merging**